packages/react-scripts/layout/views/partials/featureDetection.ejs

```diff
@@ -0,0 +1,6 @@
+const unsupportedBrowserHeader = document.getElementById('unsupported-browser');
+const outdatedFeatures = Object.keys(Modernizr).filter(feature => !Modernizr[feature]);
+
+if (outdatedFeatures.length > 0 && unsupportedBrowserHeader) {
```

Comment:
should we have some error handling in case Modernizr gets blocked or fails to load?